src/cmd/node/mod.rs

```diff
@@ -27,7 +27,7 @@ pub enum NodeCmd {
     /// Access node's RPC requests.
     #[clap(subcommand, about = "Access node's RPC requests")]
     Rpc(Rpc),
-    // TODO: Future possibility
+    // NOTE: Future possibility
```

Comment:
I've never seen any RPC which can trigger extrinsic calls. We could consider removing these comments completely.

Reply:
Done